android_mic_streaming/app/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'com.android.application'
+
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.2"
+    defaultConfig {
+        applicationId "org.mozilla.deepspeechdemo"
```

Comment:
I dont think you should namespace `org.mozilla`

Reply:
What else?

Reply:
Maybe make up something based on your github account? 😎

Reply:
I used `org.deepspeechdemo` now :sweat_smile: